Formula/python@3.7.rb

```diff
@@ -223,11 +224,6 @@ def post_install
       (libexec/"bin").install_symlink (bin/versioned_name).realpath => unversioned_name
     end
 
-    # post_install happens after link
```

Comment:
`python@3.7` will no longer be linked into the prefix, so this should be removed.

Reply:
Good point - I'll update my pull request to reflect this.